Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/OthersContentPage.xaml.cs

```diff
@@ -126,6 +129,7 @@ async void SignInInformation(object sender, EventArgs e)
         {
             if (userInfo != null)
             {
+                Application.Current.Properties[AccountId] = null;
```

Comment:
Shouldn't this be in signOut instead? I don't understand why we would lose context when looking at sign in details.